integration_tests/run-tests.sh

```diff
@@ -19,6 +19,12 @@ TESTS_EXIT_CODE=$?
 
 cp reports/testReport.xml $TEST_REPORT_ABSOLUTE_DIR/junit.xml
 
-if [ $TESTS_EXIT_CODE -ne 0 ]; then
+npm run test:ui
+
+TESTS_EXIT_CODE=$?
```

Comment:
You're using the same variable twice. You need two different ones of course.

Reply:
yes thanks fixed it